lib/helpers/getReSource.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function(re) {
+    return [re.source, [
```

Comment:
now you write in JSON:
```
['\\d\\d-\\d\\d[a-z]', 'i']
```
but in langoose you can use a regular expression `/\d\d-\d\d[a-z]/i`, this code transforms a regular expression into an array